checks.d/kafka_consumer.py

```diff
@@ -47,20 +66,37 @@ def check(self, instance):
             # Query Zookeeper for consumer offsets
             consumer_offsets = {}
             topics = defaultdict(set)
+            if not consumer_groups:
```

Comment:
This throws:

    UnboundLocalError: local variable 'consumer_groups' referenced before assignment

I fixed by changing to `if not 'consumer_groups'  in instance:` although I'd probably prefer to collapse the check together with the check in line 49 so it's only defined in one place.